src/application.rs

```diff
@@ -146,7 +146,7 @@ impl Application {
             .translator_credits(&gettext("translator-credits"))
             .modal(true)
             .authors(vec!["Marcus Behrendt".into()])
-            .artists(vec!["Marcus Behrendt".into()])
+            .artists(vec!["Marcus Behrendt".into(), "Allaeddine Boulefaat".into()])
```

Comment:
rustfmt complains. This should fix it.
```suggestion
            .artists(vec![
                "Marcus Behrendt".into(),
                "Allaeddine Boulefaat".into(),
            ])
```